ps_imageslider.php

```diff
@@ -575,6 +575,11 @@ public function getWidgetVariables($hookName = null, array $configuration = [])
 
     protected function updateUrl($link)
     {
+        // Empty link.
+        if (empty($link) || $link === '#') {
```

Comment:
what if the link is `#path-to-my-id`?
As I understand, it will generate a `http://#path-to-my-id`.

I don't get the `$link === '#'` condition :thinking:

Reply:
Good point, replaced with `strpos` fix.